packages/back-end/src/models/ExperimentModel.ts

```diff
@@ -802,4 +805,37 @@ export const logExperimentDeleted = async (
   return emittedEvent.id;
 };
 
-// endregion Events
+export const getAllVisualExperiments = async (
+  organization: string
+): Promise<
+  Array<{
+    experiment: ExperimentInterface;
+    visualChangeset: VisualChangesetInterface;
+  }>
+> => {
+  const visualChangesets = await findVisualChangesets(organization);
+  const allVisualChanges = flatten(
+    visualChangesets.map((vc) => vc.visualChanges)
+  );
+  const experiments = (
+    await ExperimentModel.find({
+      id: {
+        $in: visualChangesets.map((changeset) => changeset.experiment),
+      },
+      organization,
+      archived: false,
+    })
+  ).filter(
+    // If the experiment is stopped and the released variation doesn’t have any visual changes, it should be excluded
+    (e) =>
+      e.status !== "stopped" ||
+      allVisualChanges.some((vc) => vc.variation === e.releasedVariationId)
+  );
+  // @ts-expect-error filter boolean
```

Comment:
does this refer to e.experiment not being a bool? If so, could also do something like
`(e) => !!e.experiment`

Reply:
No .. though I added those for good measure. Typescript won't infer the filtering out of null/falsy values using the `filter` method.

Reply:
You can make typescript understand it by having a type guard function.

Reply:
Added